### ISSUE TITLE
[Problem] Dart Error: Dart_LookupLibrary: library 'package:background_fetch/background_fetch.dart' not found

### ISSUE THREAD

**Issue by Noblesoft** [transistorsoft/flutter_background_fetch#296]
---------------------------------------------------------------

__[❗ Update]__ Fixed in [`v1.1.2`](https://github.com/transistorsoft/flutter_background_fetch/blob/master/CHANGELOG.md#112---2022-10-27)
- You must now annotate your `headlessTask` with `@pragma('vm:entry-point')`

```dart
// [Android-only] This "Headless Task" is run when the Android app is terminated with `enableHeadless: true`
// Be sure to annotate your callback function to avoid issues in release mode on Flutter >= 3.3.0
@pragma('vm:entry-point')
void backgroundFetchHeadlessTask(HeadlessTask task) async {
  String taskId = task.taskId;  
  print('[BackgroundFetch] Headless event received.');
  BackgroundFetch.finish(taskId);
}
```

---------------------------------------------------------------

Hello,
How i can i solve this problem, for only in mode release (>=^1.1.0)
E/flutter (18287): [ERROR:flutter/shell/common/shell.cc(89)] Dart Error: Dart_LookupLibrary: library 'package:background_fetch/background_fetch.dart' not found.
E/flutter (18287): [ERROR:flutter/runtime/dart_isolate.cc(668)] Could not resolve main entrypoint function.
E/flutter (18287): [ERROR:flutter/runtime/dart_isolate.cc(168)] Could not run the run main Dart entrypoint.
E/flutter (18287): [ERROR:flutter/runtime/runtime_controller.cc(389)] Could not create root isolate.
E/flutter (18287): [ERROR:flutter/shell/common/shell.cc(605)] Could not launch engine with configuration.

Thank you.

**Comment by christocracy** [comment]
Ignored issue template.

**Your Environment**
* Plugin version:
* Platform: iOS or Android
* OS version:
* Device manufacturer / model:
* Flutter info (`flutter info`, `flutter doctor`):
* Plugin config

**To Reproduce**
Steps to reproduce the behavior:
1. 
2.
3.
4.

**Debug logs**
- ios XCode logs, 
- Android: `$ adb logcat`

**Additional context**
Add any other context about the problem here.

**Comment by Noblesoft** [comment]
I'm sorry
My Environment

Plugin version: ^1.1.0
Platform: Android
OS version: 9
Device manufacturer / model: OPPO CPH2083
Flutter info (flutter info, flutter doctor):

[✓] Flutter (Channel master, 3.4.0-36.0.pre.8, on Ubuntu 22.04.1 LTS 5.15.0-48-generic, locale en_US.UTF-8)                                                                      Checking Android licenses is taking an unexpectedly long time...[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Chrome - develop for the web
[✓] Linux toolchain - develop for Linux desktop
[✓] Android Studio (version 2021.2)
[✓] Android Studio (version 2021.3)
[✓] Android Studio
[✓] VS Code
[✓] Connected device (3 available)
[✓] HTTP Host Availability

Debug logs
E/flutter (18287): [ERROR:flutter/shell/common/shell.cc(89)] Dart Error: Dart_LookupLibrary: library 'package:background_fetch/background_fetch.dart' not found.
E/flutter (18287): [ERROR:flutter/runtime/dart_isolate.cc(668)] Could not resolve main entrypoint function.
E/flutter (18287): [ERROR:flutter/runtime/dart_isolate.cc(168)] Could not run the run main Dart entrypoint.
E/flutter (18287): [ERROR:flutter/runtime/runtime_controller.cc(389)] Could not create root isolate.
E/flutter (18287): [ERROR:flutter/shell/common/shell.cc(605)] Could not launch engine with configuration.

**Comment by christocracy** [comment]
>Dart Error: Dart_LookupLibrary: library 'package:background_fetch/background_fetch.dart' not found.

Your app doesn't seem to think you have this plugin installed.

>(Channel master, 3.4.0-36.0.pre.8

I note you're using a pre-release version of Flutter.

**Comment by Noblesoft** [comment]
I'm sorry again , i'm using google translate,
the problem just in the mode release, but in the mode debug the plugin is working normally.

**Comment by christocracy** [comment]
It doesn’t change my statement.  Flutter thinks that plugin doesn’t exist.

It’s flutter’s responsibility to ensure that packages in your pubspec.yaml are made available.

I suggest you clean your build.

**Comment by Noblesoft** [comment]
Honestly I tried all the solutions, the problem appeared after the flutter update. I'll try again.

Thank you very much for your reply

**Comment by cpc6128** [comment]
@Noblesoft , you have to downgrade to 3.0.5 and it's works again.

**Comment by Noblesoft** [comment]
Sorry, it didn't work?

**Comment by Noblesoft** [comment]
https://github.com/transistorsoft/flutter_background_geolocation/issues/840

**Comment by christocracy** [comment]
Fixed in [`v1.1.2`](https://github.com/transistorsoft/flutter_background_fetch/blob/master/CHANGELOG.md#112---2022-10-27)
- You must now annotate your `headlessTask` with `@pragma('vm:entry-point')`

```dart
// [Android-only] This "Headless Task" is run when the Android app is terminated with `enableHeadless: true`
// Be sure to annotate your callback function to avoid issues in release mode on Flutter >= 3.3.0
@pragma('vm:entry-point')
void backgroundFetchHeadlessTask(HeadlessTask task) async {
  String taskId = task.taskId;  
  print('[BackgroundFetch] Headless event received.');
  BackgroundFetch.finish(taskId);
}
```

**Comment by Noblesoft** [comment]
Thank you
